doc/wheel_layout.txt

```diff
@@ -1,9 +1,9 @@
 <!-- Wheels
-	/0   1\
```

Comment:
See Pull #344, I think josh fixed it when he got the 45 degree angle pull, and got reverted.